### ISSUE TITLE
[enhancement]: add controlnet-union-sdxl-1.0 supported

### ISSUE THREAD

**Issue by licyk** [invoke-ai/InvokeAI#6760]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Contact Details

licyk233@gmail.com

### What should this feature add?

need to support controlnet-union-sdxl-1.0, this controlnet encompasses a variety of control types and possesses commendable control effects.  
https://huggingface.co/xinsir/controlnet-union-sdxl-1.0

**Comment by vrubzov1957** [comment]
Yes, please. This combines the functionality of all ControlNet models in one - with it don’t need to download all ControlNet, Union is enough for most tasks that ControlNet can do
